src/App.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import carsFromServer from './api/cars';
+import colorsFromServer from './api/colors';
+import CarList from './components/CarList';
+
+// 1. Render car with color
+// 2. Add ability to filter car by brand name
+// 3. Add ability to filter car by color
+
+export const App = () => {
+  function findCollors(colorId) {
```

Comment:
```suggestion
  function findColorById(colorId) {
```
Also, would be great to return the name, not the whole color object